include/library/spdm_requester_lib.h

```diff
@@ -141,6 +141,62 @@ libspdm_return_t libspdm_get_certificate_ex(void *spdm_context,
                                             void *cert_chain,
                                             const void **trust_anchor,
                                             size_t *trust_anchor_size);
+
+/**
+ * This function sends GET_CERTIFICATE to get certificate chain in one slot from the device,
+ * and allows the Integrator to specify the size of the certificate chain blocks.
+ *
+ * This function verify the integrity of the certificate chain.
+ * root_hash -> Root certificate -> Intermediate certificate -> Leaf certificate.
+ *
+ * If the peer root certificate hash is deployed,
+ * this function also verifies the digest with the root hash in the certificate chain.
+ *
+ * @param  spdm_context     A pointer to the SPDM context.
+ * @param  session_id       Indicates if it is a secured message protected via SPDM session.
+ *                          If session_id is NULL, it is a normal message.
+ * @param  slot_id          The number of slot for the certificate chain.
+ * @param  length           The length of the certificate chain block to retrieve.
+ * @param  cert_chain_size  On input, indicate the size in bytes of the destination buffer to store the digest buffer.
+ *                          On output, indicate the size in bytes of the certificate chain.
+ * @param  cert_chain       A pointer to a destination buffer to store the certificate chain.
+ **/
+libspdm_return_t libspdm_get_certificate_choose_length(void *spdm_context,
```

Comment:
If we have `_ex` version, do we still need this non `_ex` version?

Reply:
Yes, could probably just use the `_ex` version and integrator can set trust anchor to `NULL` if they don't want it.

Reply:
@lordaule says it's ok too.

Reply:
Updated change to expose only _ex version.